### PR TITLE
Fix e2e-test on master

### LINKED ISSUE #861: Extra 0x on keccak256 calls
## 🐛 Bug Report

Since the migration from `js-sha3` to `web3.utils`, adding an extra `0x` in front of `keccak256` calls (e.g. [here](https://github.com/aragon/aragon-cli/blob/develop/packages/aragon-cli/src/commands/apm_cmds/extract-functions.js#L42)) is no longer necessary. We should remove them as the result is invalid hexadecimal (`0x0x...`).

### PR DESCRIPTION
**Pull request by 0xGabi**
# 🦅 Pull Request

Currently the CI is broke. This PR will fix that.

I have to skip two test:
1. `create-aragon-app` that test the generation of a new boilerplate environment
2. `aragon run` cause it dependent of having a environment setup with the previous test

TODO:

- [x] Fix the above test

This PR also fixes #861

### CODE REVIEW

File: packages/e2e-tests/src/cli/dao-new.test.js [hunk 1]
```diff
@@ -1,35 +0,0 @@
-import test from 'ava'
```
[the remark this answers is not in the document]
Comment (0xGabi) [reply packages/e2e-tests/src/cli/dao-new.test.js]: No, we shouldn't

[the remark this answers is not in the document]
Comment (macor161) [reply packages/e2e-tests/src/cli/dao-new.test.js]: Oh wow good catch! Thanks!

**Comment by macor161** [comment]
Is there a reason why the merging branch was changed to `develop`?

**Comment by 0xGabi** [comment]
I think @0x6431346e might misunderstood our plan with this PR. The idea is to fix e2e on master to been able to have them for stable versions. The best should be merge both on `master` and `develop` actually 😅

**Pull request merged**